clinical/src/main/kotlin/com/hartwig/actin/clinical/feed/standard/extraction/StandardPriorSequencingTestExtractor.kt

```diff
@@ -12,49 +13,60 @@ import com.hartwig.actin.datamodel.clinical.SequencedFusion
-                val mandatoryCurationTestResults = curate(ehrPatientRecord, onlyFreeTextResults)
-                val optionalCurationTestResults = curate(ehrPatientRecord, populatedResults)
-                val allResults =
-                    removeCurated(removeCurated(nonIHCTestResults, mandatoryCurationTestResults), optionalCurationTestResults) +
-                            extract(mandatoryCurationTestResults) + extract(optionalCurationTestResults)
-                if (allResults.isNotEmpty()) {
-                    ExtractionResult(
-                        listOf(
-                            PriorSequencingTest(
-                                test = test.test,
-                                date = test.date,
-                                variants = variants(allResults),
-                                fusions = fusions(allResults),
-                                amplifications = amplifications(allResults),
-                                skippedExons = skippedExons(allResults),
-                                deletedGenes = geneDeletions(allResults),
-                                isMicrosatelliteUnstable = msi(allResults),
-                                tumorMutationalBurden = tmb(allResults),
-                                testedGenes = (test.testedGenes ?: emptySet()) + impliedTestedGenes(allResults)
-                            )
-                        ),
-                        mandatoryCurationTestResults.map { curated -> curated.extractionEvaluation }
-                            .fold(CurationExtractionEvaluation()) { acc, extraction -> acc + extraction }
-                    )
-                } else null
-            } else {
-                null
-            }
+            val testCurationConfig =
+                CurationResponse.createFromConfigs(
+                    testCuration.find(test.test),
+                    ehrPatientRecord.patientDetails.hashedId,
+                    CurationCategory.SEQUENCING_TEST,
+                    test.test,
+                    "sequencing test",
+                    false
+                )
+            testCurationConfig.config()?.let { testCuration ->
+                if (!testCuration.ignore) {
+                    val nonIHCTestResults = test.results.filter { it.ihcResult == null }.toSet()
+                    val (onlyFreeTextResults, populatedResults) = nonIHCTestResults
+                        .partition { checkAllFieldsNull(it) }
+                    val mandatoryCurationTestResults = curate(ehrPatientRecord, onlyFreeTextResults)
+                    val optionalCurationTestResults = curate(ehrPatientRecord, populatedResults)
+                    val allResults =
+                        removeCurated(removeCurated(nonIHCTestResults, mandatoryCurationTestResults), optionalCurationTestResults) +
+                                extract(mandatoryCurationTestResults) + extract(optionalCurationTestResults)
```

Comment:
Same as:
```suggestion
                                extract(mandatoryCurationTestResults + optionalCurationTestResults)
```
? It seems like you could combine these lists above to simplify this step.